### PR TITLE
Index html again

### PR DESCRIPTION
**Pull request by atd-schubert**
Menu for the index!

### CODE REVIEW

File: examples/property.component.ts [hunk 1]
```diff
@@ -98,10 +98,10 @@ export class ExamplePropertiesComponent implements AfterViewInit {
     <div class="container">
         <ul class="nav navbar-nav">
             <a class="navbar-brand" href="https://yagajs.org" data-placement="bottom" title="YAGA">YAGA</a>
-			<li class="nav-item nav-link"><a href="https://leaflet-ng2.yagajs.org" title="leaflet-ng2"><span class="fa fa-home" aria-hidden="true"></span>  leaflet-ng2</a></li>
-			<li class="nav-item nav-link"><a href="../../" title="Last release"><span class="fa fa-cubes" aria-hidden="true"></span>  Release</a></li>
+            <li class="nav-item nav-link"><a href="https://leaflet-ng2.yagajs.org" title="Home"><span class="fa fa-cube" aria-hidden="true"></span>  leaflet-ng2</a></li>
```
Comment (OpenSteveMap) [inline examples/property.component.ts]: Why `title="Home"` ?

File: index.html [hunk 2]
```diff
@@ -7,14 +7,19 @@
...
 <body>
 <header id="top" class="navbar navbar-default navbar-static-top navbar-inverse">
     <div class="container">
         <ul class="nav navbar-nav">
             <a class="navbar-brand" href="https://yagajs.org" data-placement="bottom" title="YAGA">YAGA</a>
+<<<<<<< HEAD
+            <li class="nav-item nav-link"><a href="https://leaflet-ng2.yagajs.org" title="Home"><span class="fa fa-cube" aria-hidden="true"></span>  leaflet-ng2</a></li>
+=======
             <li class="nav-item nav-link"><a href="http://leaflet-ng2.yagajs.org" title="leaflet-ng2"><span class="fa fa-cube" aria-hidden="true"></span>  leaflet-ng2</a></li>
+>>>>>>> develop
```
Comment (OpenSteveMap) [inline index.html]: The merge conflict should be removed here

File: index.html [hunk 3]
```diff
@@ -7,14 +7,19 @@
...
             <li class="nav-item nav-link active"><a href="#" title="Last release"><span class="fa fa-cubes" aria-hidden="true"></span>  Release</a></li>
```
Comment (OpenSteveMap) [inline index.html]: Do we really want `fa-cubes` as icon for the latest release?

File: examples/index.html [hunk 4]
```diff
@@ -22,12 +23,22 @@
...
+    <ul class="list-group">
         <li class="list-group-item"><a href="./tile-layer-directive">Tile-Layer</a></li>
+    </ul>
+
+    <h4>Vector-Layer</h4>
+    <ul class="list-group">
         <li class="list-group-item"><a href="./circle-marker-directive">Circle-Marker-Directive</a></li>
     </ul>
+
     <h4>Control examples</h4>
```
Comment (OpenSteveMap) [inline examples/index.html]: We should remove examples here or we should add examples to the headings above.

File: examples/property.component.ts (hunk 1 above)
Comment (OpenSteveMap) [inline examples/property.component.ts]: Why `title="Home"`?

File: index.html (hunk 2 above)
Comment (OpenSteveMap) [inline index.html]: We should remove the lines from the merge-conflict:
`+<<<<<<< HEAD`

`+=======`

`+>>>>>>> develop`

File: index.html (hunk 3 above)
Comment (OpenSteveMap) [inline index.html]: Do we really wanna use here `fa-cubes` for the latest release?

File: examples/property.component.ts [hunk 5]
```diff
@@ -98,10 +98,10 @@ export class ExamplePropertiesComponent implements AfterViewInit {
...
+            <li class="nav-item nav-link"><a href="../../" title="Last release"><span class="fa fa-cubes" aria-hidden="true"></span>  Release</a></li>
```
Comment (OpenSteveMap) [inline examples/property.component.ts]: Do we really wanna use here `fa-cubes`?

**Review by OpenSteveMap — COMMENTED** [review]
See the comments below:

**Comment by OpenSteveMap** [comment]
@atd-schubert should i change the comments? We should not do it both at the same time.

File: examples/property.component.ts (hunk 5 above)
Comment (OpenSteveMap) [reply examples/property.component.ts]: maybe `fa-flag` for release?

File: index.html (hunk 3 above)
Comment (OpenSteveMap) [reply index.html]: maybe `fa-flag`?

**Comment by atd-schubert** [comment]
You can do it!

File: index.html (hunk 3 above)
Comment (atd-schubert) [reply index.html]: `fa-circle-check-o`?

Comment (atd-schubert) [reply index.html]: Or `fa-tags`?

**Review by OpenSteveMap — APPROVED** [review]
should be fine now

**Pull request merged**